sections/reviews.md

```diff
@@ -0,0 +1,74 @@
+# Code reviews
+
+At Open Knowledge International (OKI) all code must be reviewed by a colleague before merging into the main branch of the repository (usually `master`). This short document examples how to do code reviews, and why we do it.
+
+## How
+
+The process for code reviews is straight forward as long as you are already following our coding standards for working with version control, and in particular our git flow-influenced process of branch management. The bottom line is that, before any code hits `master`, it must be reviewed by a colleague.
+
+### Submitting code for review
+
+Submitting code for review should follow these steps:
+
+1. Work out of your own fork of the main repository (usually `origin`).
```

Comment:
@brew yes that is the main annoying thing. In the last year I removed something like 20 branches from both OpenSpending and Open Data Census because people just merged PRs and left their mess around :)